feature/interface/singleton/ate_tests/singleton_test/README.md

```diff
@@ -0,0 +1,90 @@
+# RT-5.1: Singleton Interface
+
+## Summary
+
+Singleton L3 interface (non-LAG) is supported on DUT.
+
+## Procedure
+
+*   For each port speed and breakout port configuration that need to be tested,
```

Comment:
Dedent everything by one level? Namely, "for each port speed" will not be a bullet point, and "Configure ..." will be level 1, etc.

Reply:
Done!